backend/src/test/java/pl/project/plannerapp/testexample/CampOrganisationTest.java

```diff
@@ -87,13 +89,15 @@ public void shouldThrowExceptionWhenOnePartOfNameIsNull() {
         //given
         //when
         //then
-        Assertions.assertThrows(IllegalArgumentException.class, () -> {
+        Throwable exception = assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
`Throwable` is the highest class in the hierarchy here. We want to test if it's exactly `IllegalArgumentException`. We are checking it in `assertThrows` but we can add a double check by expecting here not `Throwable` but `IllegalArgumentException` :)